src/components/Feed/Feed.tsx

```diff
@@ -20,6 +20,8 @@ import DialogTitle from '@material-ui/core/DialogTitle';
 import Post from './Post';
 import Rockstar from './Rockstar';
 
+import RecognitionService from '../../api/RecognitionService';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Benedict-Scrumberbatch/peer-recognition/pullRequest?prid=7204339)
```suggestion
import RecognitionService from "../../api/RecognitionService";
```